server/server.go

```diff
@@ -128,6 +131,10 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 			return nil, err
 		}
 	}
+
+	//
+	s.sqlDB = sqlserver.NewDBServer(s.db)
```

Comment:
Maybe we're being too aggressive with subpackaging too early. I think anything which needs to be exported outside of sql/ should maybe just be in sql directly.